BCStudentSoftwareDevTeam/celts
# Add RSVP history, limit, and waitlist

## Resolves issue #829 and #748 

# Did: 
## For Issue #748:
 
Added a new column to the Event table `rsvpLimit` and created a function `getCurrentRsvpAmmount()` in events.py to get the amount of people that have RSVPd to an event. The RSVP button is enabled/disabled based off the current RSVP amount. The wording on the Event View page now changes based off of whether or not RSVP is required, if there is a limit or not, and how many spots are left if there is a limit. 

## For Issue #829: 

# To Test:
- Checkout the branch `limitRsvp`
- Reset your database with `database/reset_database.sh test`
- Run the test suite with `tests/run_tests.sh ` and verify that all tests pass
- Run the application with `flask run` 
- Create multiple events, some that require RSVP but have no limit and some that require RSVP with a limit
- Verify that all the behavior that you encounter is as you expect
